### DISCUSSION TITLE
Add TheSportsDB scraper to TV scraper list

### DISCUSSION THREAD

**Discussion by pneshati** [Komet/MediaElch#1499] [Ideas]
I would love the option to scrape sport events within MediaElch by using the existing TheSportsDB kodi scraper.
A few potential issues below, however;

- multiple events can share the episode numbers so the naming convention works better in the '.special' format
- the following fields won't apply: director, writer
- athletes would be sorted as actors per TheSportsDB API

SportsDB scraper on Git:
https://github.com/zag2me/metadata.thesportsdb.com

SportsDB API guide:
https://www.thesportsdb.com/api.php

**Comment by bugwelle** [comment]
Hi, cool idea 👍  :)

It would be great if you could provide some examples, so that I can integrate them into our test as well.  For example: What file-names (and episode names/patterns) are common? How would the "special" format look like in your case?

Regards,
Andre
